src/open_inwoner/qmatic/client.py

```diff
@@ -85,7 +85,7 @@ class NoServiceConfigured(RuntimeError):
 # API CLIENT IMPLEMENTATIONS, per major version of the API
 
 
-def QmaticClient() -> "Client":
+def make_qmatic_client() -> "Client":
```

Comment:
I think this is more in line with what we use elsewhere (e.g. for zgw clients):

```suggestion
def qmatic_client_factory() -> "Client":
```